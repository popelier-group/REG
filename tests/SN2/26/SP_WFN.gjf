# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.39058500   -0.00003100    0.00006000
 H                 -1.06622200   -0.87557800   -0.54565100
 H                 -1.06621700    0.91034900   -0.48531600
 H                 -1.06629000   -0.03486700    1.03118300
 Br                 1.69238200    0.00000600   -0.00000400
 Cl                -3.25001100   -0.00000100   -0.00001800

frame.wfn
