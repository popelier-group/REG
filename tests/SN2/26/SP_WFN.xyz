
6

C        1.263643    0.006071    0.087251
H        1.002981   -0.540886   -0.808566
H        0.879630   -0.481138    0.973116
H        0.937616    1.035654    0.030152
Br       -1.811964   -0.008629   -0.125145
Cl        3.118627    0.014821    0.215405
