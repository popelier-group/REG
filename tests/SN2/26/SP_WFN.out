 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.26/Gau-111617.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.26/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=    111618.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.39059  -0.00003   0.00006 
 H                    -1.06622  -0.87558  -0.54565 
 H                    -1.06622   0.91035  -0.48532 
 H                    -1.06629  -0.03487   1.03118 
 Br                    1.69238   0.00001  0. 
 Cl                   -3.25001  0.        -0.00002 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.390585   -0.000031    0.000060
      2          1           0       -1.066222   -0.875578   -0.545651
      3          1           0       -1.066217    0.910349   -0.485316
      4          1           0       -1.066290   -0.034867    1.031183
      5         35           0        1.692382    0.000006   -0.000004
      6         17           0       -3.250011   -0.000001   -0.000018
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.081478   0.000000
     3  H    1.081479   1.786946   0.000000
     4  H    1.081478   1.786953   1.786953   0.000000
     5  Br   3.082967   2.945212   2.945186   2.945307   0.000000
     6  Cl   1.859426   2.415219   2.415203   2.415208   4.942393
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        1.263643    0.006071    0.087251
      2          1           0        1.002981   -0.540886   -0.808566
      3          1           0        0.879630   -0.481138    0.973116
      4          1           0        0.937616    1.035654    0.030152
      5         35           0       -1.811964   -0.008629   -0.125145
      6         17           0        3.118627    0.014821    0.215405
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.0387749           0.8103612           0.8103612
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       168.5157248094 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.65D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40180454     A.U. after   11 cycles
            NFock= 11  Conv=0.70D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55543-101.40830 -62.20849 -56.02629 -56.02622
 Alpha  occ. eigenvalues --  -56.02622 -10.08167  -9.32260  -8.41724  -7.08627
 Alpha  occ. eigenvalues --   -7.07785  -7.07785  -6.24032  -6.23996  -6.23996
 Alpha  occ. eigenvalues --   -2.35271  -2.35259  -2.35259  -2.35230  -2.35230
 Alpha  occ. eigenvalues --   -0.70797  -0.54233  -0.48866  -0.30628  -0.30628
 Alpha  occ. eigenvalues --   -0.27071  -0.17447  -0.17447  -0.05399  -0.05312
 Alpha  occ. eigenvalues --   -0.05312
 Alpha virt. eigenvalues --    0.10998   0.12438   0.14884   0.14884   0.17487
 Alpha virt. eigenvalues --    0.18421   0.19574   0.19574   0.20737   0.20738
 Alpha virt. eigenvalues --    0.21725   0.22496   0.22496   0.24826   0.28420
 Alpha virt. eigenvalues --    0.30818   0.30819   0.36807   0.40519   0.53609
 Alpha virt. eigenvalues --    0.53609   0.55395   0.58616   0.58616   0.60074
 Alpha virt. eigenvalues --    0.62080   0.62080   0.65179   0.76655   0.76655
 Alpha virt. eigenvalues --    0.79521   0.84303   0.87872   0.87872   0.89222
 Alpha virt. eigenvalues --    0.89223   0.95416   0.96953   0.96953   1.01211
 Alpha virt. eigenvalues --    1.01211   1.03613   1.09823   1.26508   1.26508
 Alpha virt. eigenvalues --    1.26877   1.57003   1.67742   1.67742   1.85239
 Alpha virt. eigenvalues --    1.85239   1.87560   2.09129   2.21359   2.21361
 Alpha virt. eigenvalues --    2.27662   2.27662   2.30053   2.45717   2.45717
 Alpha virt. eigenvalues --    2.57286   2.59720   2.64160   2.64161   2.67308
 Alpha virt. eigenvalues --    2.67308   3.12028   3.29070   3.87270   3.87270
 Alpha virt. eigenvalues --    4.39610   4.39612   4.40406   4.40407   4.52591
 Alpha virt. eigenvalues --    6.76577   7.80367   7.80371   7.86240   9.84175
 Alpha virt. eigenvalues --   24.03542  25.98873  25.98873  26.49308  48.07925
 Alpha virt. eigenvalues --  215.82755 290.01217 290.01217 290.065541020.83768
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.676705   0.404831   0.404830   0.404834  -0.061291  -0.175236
     2  H    0.404831   0.520411  -0.038403  -0.038404  -0.000004  -0.041267
     3  H    0.404830  -0.038403   0.520410  -0.038403  -0.000007  -0.041266
     4  H    0.404834  -0.038404  -0.038403   0.520423  -0.000002  -0.041277
     5  Br  -0.061291  -0.000004  -0.000007  -0.000002  35.923496   0.056982
     6  Cl  -0.175236  -0.041267  -0.041266  -0.041277   0.056982  17.246722
 Mulliken charges:
               1
     1  C   -0.654673
     2  H    0.192836
     3  H    0.192839
     4  H    0.192829
     5  Br  -0.919173
     6  Cl  -0.004658
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.076169
     5  Br  -0.919173
     6  Cl  -0.004658
 Electronic spatial extent (au):  <R**2>=           1183.1293
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              4.3195    Y=              0.0207    Z=              0.2983  Tot=              4.3298
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -75.4384   YY=            -44.4171   ZZ=            -44.5643
   XY=             -0.1475   XZ=             -2.1427   YZ=             -0.0102
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -20.6318   YY=             10.3895   ZZ=             10.2422
   XY=             -0.1475   XZ=             -2.1427   YZ=             -0.0102
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            -46.1434  YYY=              0.7430  ZZZ=             -1.7009  XYY=             -8.5967
  XXY=             -0.1417  XXZ=             -1.9994  XZZ=             -8.6973  YZZ=             -0.9029
  YYZ=             -0.6811  XYZ=              0.0532
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1578.9925 YYYY=            -76.1143 ZZZZ=            -82.5640 XXXY=             -3.9290
 XXXZ=            -56.8656 YYYX=             -2.5069 YYYZ=             -0.1742 ZZZX=            -46.9104
 ZZZY=             -0.3729 XXYY=           -251.8919 XXZZ=           -253.6564 YYZZ=            -26.4687
 XXYZ=             -0.0213 YYXZ=            -15.7182 ZZXY=             -1.7973
 N-N= 1.685157248094D+02 E-N=-7.690424452606D+03  KE= 3.070045758145D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE539\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.390585,-0.000031,0.00006\H,0,-1.066222,-0.875578,-0.545651\H,0,
 -1.066217,0.910349,-0.485316\H,0,-1.06629,-0.034867,1.031183\Br,0,1.69
 2382,0.000006,-0.000004\Cl,0,-3.250011,-0.000001,-0.000018\\Version=ES
 64L-G16RevC.01\State=1-A\HF=-3074.4018045\RMSD=6.986e-09\Dipole=-1.703
 49,-0.0000231,0.0000464\Quadrupole=-15.4497779,7.7248859,7.7248919,-0.
 0000338,-0.0001003,0.0000087\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 MUD THROWN IS GROUND LOST -- FROM A TEA BAG
 Job cpu time:       0 days  0 hours  0 minutes 28.4 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.5 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:54:06 2020.
